app/grandchallenge/reader_studies/views.py

```diff
@@ -486,18 +475,34 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class AddGroundTruthToReaderStudy(BaseAddObjectToReaderStudyMixin, FormView):
-    form_class = GroundTruthForm
-    template_name = "reader_studies/readerstudy_add_object.html"
-    type_to_add = "ground truth"
+class AddGroundTruthViaCSVToReaderStudy(
+    BaseAddObjectToReaderStudyMixin, FormView
+):
+    form_class = GroundTruthCSVForm
+    template_name = "reader_studies/ground_truth_via_csv.html"
```

Comment:
This template name is not constant with the rest of the application. Maybe something like `ground_truth_csv_form.html`?

Reply:
I agree, that makes it better suite the rest of the naming scheme.